### ISSUE TITLE
Cannot finish Quest: The Sand Charm

### ISSUE THREAD

**Issue by Fiocitrine** [DarkstarProject/darkstar#753]
I cannot finish quest because NPC Celestina (https://github.com/DarkstarProject/darkstar/blob/master/scripts/zones/Mhaura/npcs/Celestina.lua) needs rare/ex item Sand Charm (ID 13095) that drops from Crossbones (ID 17707020), which do not spawn on the Mhaura/Selbina Ferry.

**Comment by gedads** [comment]
yes... there's 2 area (actually 4) mhaura - selbina 2 normal 2 with pirates attack.
Pirate attacks isn't working so... ni pirates no drop no quest

**Comment by Fiocitrine** [comment]
I checked the sql and found both the mob and the item in database. Is there a way to just make the mob spawn on the boat?

**Comment by gedads** [comment]
DSP's goal is to mimick retail, making the pirates spawns is more like a custom feature.
retail choose the zone you're sent in (pirates or not pirates) then the pirate ship comes, 3 npcs come out from behind crates and start summoning mobs -> the pirates and eventually, they'll summon the nm.

**Comment by Fiocitrine** [comment]
I understand that DSP's goal is to mimick retail. I'm not looking for short ways around. But, scripting the pirate fight is way beyond my scope and knowledge. I figured something is better than nothing, until it's fully scripted.

Cheers

**Comment by Scavenge** [comment]
Musashi on our server was playing with the pirate attacks on his own time and was able to get almost the entire process to work, except for the mages that stand on the opposite boat and summon the skeletons.  he said it's all in there already he just had to trigger them, and i think the mages could be done by using npcs that have hidden names and untargetable.  but priorities may not lie in pirate attacks at the moment.

**Comment by Fiocitrine** [comment]
I just wanted it to be able to finish this quest and move onto the next quest, which seems fully scripted (weird).

**Comment by JoeCT** [comment]
@Scavenge - Any chance of my getting hands on the scripts you have for the pirate attack?  I'd like to take a look at them.
